docs/semgrep-secrets/conceptual-overview.md

```diff
@@ -10,254 +10,149 @@ tags:
 
 import MoreHelp from "/src/components/MoreHelp"
 
-# Conceptual overview of Semgrep Secrets
+# Semgrep Secrets overview
 
-**Semgrep Secrets** scans code to detect exposed API keys, passwords, and other credentials. These exposed secrets can be used by malicious actors to leak data or gain access to sensitive systems. Semgrep Secrets enables the user to know the following: 
+**Semgrep Secrets** scans code to detect exposed API keys, passwords, and other
+credentials. When exposed, these can be used by malicious actors to leak data
+or gain access to sensitive systems. Semgrep Secrets allows you to determine:
 
 * What secrets have been committed to your repository.
-* The validation status of the secret, such as if it is confirmed valid. **Valid** secrets are secrets that are tested against a web service and confirmed to successfully grant resources or authentication. They are actively in use. 
-* For GitHub repositories: If these secrets are in public or private repositories.
+* The validation status of the secret; for example, **valid** secrets are those that have been tested against a web service and
+confirmed to successfully grant resources or authentication. They are actively
+in use. 
+* For GitHub repositories: if there are credentials in public or private repositories.
 
-Save time and effort by prioritizing valid leaked secrets. Inform developers of valid secrets in their GitHub PRs by posting PR comments. 
+Save time and effort by prioritizing valid leaked secrets. Inform developers of valid secrets in their PRs and MRs by posting comments directly. 
```

Comment:
```suggestion
Save time and effort by prioritizing valid leaked secrets. Inform developers of valid secrets in their PRs and MRs by posting comments directly. 
```
Let's look at the actors:
* Save time and effort by prioritizing... > the security engineer does this
* Inform developers of valid secrets > Semgrep does this for security engineers.

It could sound like security engineers still have to post the comments. Could you update this please?